engine/storage/datamotion/src/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -114,6 +142,17 @@ private boolean canHandle(DataStore dataStore) {
 
                     return true;
                 }
+            } else if (dataObject instanceof TemplateInfo) {
+                // If the storage system can clone volumes, we can cache templates on it.
+                String value = mapCapabilities.get(DataStoreCapabilities.CAN_CREATE_VOLUME_FROM_VOLUME.toString());
```

Comment:
Yes, and value equaling null will get converted into a boolean of false later (only "true" (ignoring case) will result in a true boolean).